### PR TITLE
Added Contributor Count

### LINKED ISSUE #1028: Add contributors count to tag pages
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

The new tag pages have a great "intro" section sourced from the first paragraph of the corresponding wiki page, as of #985: 

https://publiclab.org/tag/dustduino

But if no corresponding page exists, they do not.

You can see the contributor count for that topic here:

https://publiclab.org/contributors/dustduino

screen shot 2016-11-28 at 3 19 43 pm

We should show how many people contributed to that topic on the tag page itself, just below the title, in smaller grey text just below this line: https://github.com/publiclab/plots2/blob/master/app/views/tag/show.html.erb#L12

See how we calculate that in the `contributors` action in the tag_controller: https://github.com/publiclab/plots2/blob/master/app/controllers/tag_controller.rb#L217

```ruby
    @users = @notes.collect(&:author).uniq
```

However, that `@notes` is *all* notes, not just those on this page (since /tag/dustuino is paginated). So see the tag_controller to see how that's calculated. We'll likely have to add that code as well to be able to display this on the main tag page under `def show` in the same controller.

**Update:** We may just want to display the number in the tab for Contributors, just before the word, like: `14 Contributors`. That would go on this line:

https://github.com/publiclab/plots2/blob/master/app/views/tag/show.html.erb#L61 And could look like this:

```ruby
     <li<% if @node_type == "contributors" %> class="active"<% end %>><a href="/contributors/<%= params[:id] %>"><i class="fa fa-user"></i> <%= @notes.collect(&:author).uniq.length %> <%= raw t('tag.show.contributors') %></a></li>
```

However, we still want to be sure we're counting **all** contributors, and not just the ones for content shown on this page. We may want to add a followup issue to convert this count to a generalized tag-based method for counting contributors. OK -- did that in #1419

### PR DESCRIPTION
**Pull request by devaman**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #1028  reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software
Fixes #1028 
We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
oh cool! awesome. I hope to take a look at this tomorrow morning, thank you
and thanks for your patience!!

File: app/controllers/tag_controller.rb [hunk 1]
```diff
@@ -55,6 +55,15 @@ def show
     @wikis = nodes if @node_type == 'wiki'
     @nodes = nodes if @node_type == 'maps'
     @title = params[:id]
+    notes = Node.where(status: 1, type: 'note')
+                 .includes(:revision, :tag)
+                 .where('term_data.name = ?', params[:id])
+    users = notes.collect(&:author).uniq
+    if !users.first.nil?
```
Comment (jywarren) [inline app/controllers/tag_controller.rb]: Hmm, i wonder if we could do this more compactly like:

 `@length = user.length || 0` ?

does that syntax work in Ruby?

Thanks for your contribution!!

**Comment by jywarren** [comment]
Also, could you add "Fixes #____" with the # of the issue, to the description? that way when we merge it, i'll automatically close the issue. Thanks a million!

**Comment by devaman** [comment]
ok .. Done with changes!

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. If you're looking for a new challenge, please take a look at our `help-wanted` list: https://github.com/publiclab/plots2/labels/help-wanted

Thanks!

**Pull request merged**